pyfstat/mcmc_based_searches.py

```diff
@@ -235,8 +235,8 @@ def _get_search_ranges(self):
                 )
             # first start with parameters that have non-delta prior ranges
             search_ranges = {
-                key: [prior_bounds[key]["lower"], prior_bounds[key]["upper"], 0]
```

Comment:
Thanks, you've found the source of the problem, but I think the right solution is to just drop that third entry in the list - `ComputeFstat.estimate_min_max_CoverFreq()` will accept `either [single_value], [min,max] or [min,max,step]` and so you're now basically telling it the MCMC has a "step size" of the whole covered band. But anyway the step size is not needed for the coverage band estimation, so we can just fall back on the default.

I'll push a new version.